frontend/components/common/other/FeaturesView.vue

```diff
@@ -12,7 +12,7 @@
 <script lang="ts">
 import Vue from 'vue';
 import LandingBlock from '~/components/layout/LandingBlock.vue';
-import FeatureList from '~/components/FeatureList.vue';
+import FeatureList from '~/components/common/other/FeatureList.vue';
```

Comment:
I think `FeatureList` is a part of the unified page. Can you please assemble all related components under a separate `unified` folder? (sorry if I missed this in the proposal)

Reply:
FeatureList is actually not a part of the unified page. It is part of `/features`. However I will still make the unified folder for the unified components 😉